File: data/code_pairs/fortran/1082.f90
module FoX_wcml
  implicit none
contains

  subroutine cmlBeginFile()
    print *, "Beginning file..."
  end subroutine cmlBeginFile

  subroutine cmlFinishFile()
    print *, "Finishing file..."
  end subroutine cmlFinishFile

  subroutine cmlAddNamespace()
    print *, "Adding namespace..."
  end subroutine cmlAddNamespace

  subroutine cmlStartCml()
    print *, "Starting CML..."
  end subroutine cmlStartCml

  subroutine cmlEndCml()
    print *, "Ending CML..."
  end subroutine cmlEndCml

end module FoX_wcml

File: data/code_pairs/cpp/1082.cpp
#include <iostream>

class FoX_wcml {
public:
    void cmlBeginFile() {
        std::cout << "Beginning file..." << std::endl;
    }

    void cmlFinishFile() {
        std::cout << "Finishing file..." << std::endl;
    }

    void cmlAddNamespace() {
        std::cout << "Adding namespace..." << std::endl;
    }

    void cmlStartCml() {
        std::cout << "Starting CML..." << std::endl;
    }

    void cmlEndCml() {
        std::cout << "Ending CML..." << std::endl;
    }
};